.circleci/config.yml

```diff
@@ -16,6 +16,12 @@ commands:
           command: bazel run @graknlabs_build_tools//ci:run-bazel-rbe -- << parameters.command >>
           no_output_timeout: 1h
 
+  run-grakn-server:
+    steps:
+      - run: bazel build @graknlabs_grakn_core//:assemble-linux-targz
```

Comment:
We can run this line with RBE, because RBE would have the cache of Grakn build dependencies, and it will simply copy it over to the local machine into `bazel-genfiles/`